examples/chart/teleport-kube-agent/README.md

```diff
@@ -8,7 +8,8 @@ with an existing Teleport cluster:
 
 To use it, you will need:
 - an existing Teleport cluster (at least proxy and auth services)
-- a reachable proxy endpoint (`$PROXY_ENDPOINT`)
+- a reachable proxy endpoint (`$PROXY_ENDPOINT`, Ex: https://teleport.example.com:3080) 
```

Comment:
Remove the `https` as Teleport handles this automatically, also add `:443` as an example because it's becoming more common for people to use the HTTPS port.

```suggestion
- a reachable proxy endpoint (`$PROXY_ENDPOINT` e.g. `teleport.example.com:3080` or `teleport.example.com:443`) 
```